doc/build-instructions/Build_Instructions_V11.md

```diff
@@ -499,7 +499,7 @@ The following instructions guide you through the process of building a macOS **O
 You must install a number of software dependencies to create a suitable build environment on your system:
 
 - [Xcode 9.4]( https://developer.apple.com/download/more/) (requires an Apple account to log in).
-- [macOS OpenJDK 10](https://adoptopenjdk.net/archive.html?variant=openjdk10&jvmVariant=hotspot), which is used as the boot JDK.
+- [macOS OpenJDK 11](https://api.adoptopenjdk.net/v3/binary/latest/11/ga/windows/x64/jdk/hotspot/normal/adoptopenjdk), which is used as the boot JDK.
```

Comment:
Why still 'hotspot' here?

Reply:
If you can tell me that OpenJ9 works to build, then I will change it.

Reply:
I don't remember using anything other than an OpenJ9 boot jdk to build Java 11+.

Reply:
Confirmed on the OpenJ9 build machine as well. Switched to OpenJ9, and a mac download.

Reply:
Wow! I'm surprised I missed the 'windows' part of that URL.